mysql-test/suite/innodb/t/percona_force_encryption.test

```diff
@@ -0,0 +1,93 @@
+-- source include/have_innodb.inc
+-- source include/not_embedded.inc
+-- source include/have_partition.inc
```

Comment:
Partitioning is not used in the testcase - adding such tests might be a good idea